FirebaseVertexAI/Sources/Types/Public/Schema.swift

```diff
@@ -232,12 +279,15 @@ public final class Schema: Sendable {
   ///     formats ``IntegerFormat/int32`` and ``IntegerFormat/int64`` are supported; custom values
   ///     may be specified using ``IntegerFormat/custom(_:)`` but may be ignored by the model.
   public static func integer(description: String? = nil, nullable: Bool = false,
-                             format: IntegerFormat? = nil) -> Schema {
+                             format: IntegerFormat? = nil,
+                             minimum: Int? = nil, maximum: Int? = nil) -> Schema {
     return self.init(
       type: .integer,
       format: format?.rawValue,
       description: description,
-      nullable: nullable
+      nullable: nullable.self,
+      minimum: minimum != nil ? Double(minimum!) : nil,
+      maximum: maximum != nil ? Double(maximum!) : nil
```

Comment:
nit: Consider using `map` instead of the ternary.

```suggestion
      minimum: minimum.map { Double($0) },
      maximum: maximum.map { Double($0) }
```